### PR TITLE
support connection urls

### PR DESCRIPTION
**Pull request by simonstre**
[Review on Reviewable](https://reviewable.io/reviews/bluehotdog/sails-migrations/51)

### CODE REVIEW

File: lib/sails-migrations/helpers/config_loader.js [hunk 1]
```diff
@@ -51,6 +45,20 @@ function getConfigFromSailsConfig(sailsConfig) {
   return result;
 }
 
+function getConnectionFromSailsConfig(sailsConfig) {
+  if (typeof sailsConfig.defaultAdapter.config.url !== 'undefined' && sailsConfig.defaultAdapter.config.url !== null) {
+    return sailsConfig.defaultAdapter.config.url;
```
Comment (BlueHotDog) [inline lib/sails-migrations/helpers/config_loader.js]: looks cool, but could you explain a bit more what the purpose of this? why and how this works?

thanks

Comment (simonstre) [inline lib/sails-migrations/helpers/config_loader.js]: In knex, you have two options to create the db connection. One option is to pass the parameters in a object and the other one is to specify the connection url (http://knexjs.org/#Installation-client). If specified, the connection url should contains all the required information to connect to the database. In my sails app, I use the db connection url as given by heroku.

**Comment by simonstre** [comment]
If you merge this pull request, will you be able to create a new release? Thank you.

**Comment by BlueHotDog** [comment]
Sure thing, will do tomorrow!

**Pull request merged**